### PR TITLE
feat(docs): add a VisuallyHidden 'Skip to main content' element

### PR DESCRIPTION
**Pull request by clairehhelms**
## What I did
Add a visually hidden jump to content section on the docs site that shows when a user tabs into the site. If selected, the user's keyboard focus will jump to the page's content. This is super useful because currently, those interacting with the docs site via keyboard have to tab through all top and side navigation before reaching main content.

Screen Shot 2022-09-02 at 1 09 48 PM

### CODE REVIEW

File: build.washingtonpost.com/components/NavigationBar.js [hunk 1]
```diff
@@ -126,6 +133,22 @@ export const NavigationBar = ({ setMobileMenu, mobileMenuState, isClosed }) => {
   return (
     <>
       <Container>
+        <VisuallyHidden
```
Comment (artmsilva) [inline build.washingtonpost.com/components/NavigationBar.js:136]: Can we write this as a styled component?

```
const SkipToMainContent = styled(VisuallyHidden, {
  color: theme.colors.cta,
  left: "$050",
  "&:active": {
    position: "relative",
  },
  "&:focus": {
    position: "relative",
  }
}
```

Comment (clairehhelms) [reply build.washingtonpost.com/components/NavigationBar.js:136]: yessir

**Review by artmsilva — APPROVED** [review]
LGTM! I left a suggestion. Not required.

**Review by wp-aberg — APPROVED** [review]
Looks good!

**Review by atomictangerine — APPROVED** [review]

**Pull request merged**